File: 프로그래머스/0/120813. 짝수는 싫어요/짝수는 싫어요.java
import java.util.stream.*;

class Solution {
    public int[] solution(int n) {
        
        return IntStream.rangeClosed(0, n)
                        .filter(i -> i % 2 == 1)
                        .toArray();
    }
}

// 다른 풀이
int cnt = 0;

if(n % 2 == 0) cnt = n/2;
else cnt = n/2 + 1;

int[] result = new int[cnt];
int num = 1;

for(int i=0; i<result.length; i++){
    result[i] = num;
    num = num + 2;
}

return result;

File: 프로그래머스/0/120813. 짝수는 싫어요/짝수는 싫어요.py
def solution(n):
    answer = []
    
    for i in range(1, n + 1):
        if(i % 2 == 1):
            answer.append(i)
    
    return answer

// 다른 풀이
return [i for i in range(1, n+1, 2)]

// 다른 풀이
return [x for x in range(n + 1) if x % 2]
